components/alert/src/tokens.json

```diff
@@ -89,6 +97,44 @@
           }
         }
       },
+      "info": {
+        "background-color": {
+          "$extensions": {
+            "nl.nldesignsystem.css.property": {
+              "syntax": "<color>",
+              "inherits": true
+            },
+            "nl.nldesignsystem.fallback": ["utrecht.alert.background-color"]
+          }
+        },
+        "color": {
+          "$extensions": {
+            "nl.nldesignsystem.css.property": {
+              "syntax": "<color>",
+              "inherits": true
+            },
+            "nl.nldesignsystem.fallback": ["utrecht.alert.color"]
+          }
+        },
+        "border-color": {
+          "$extensions": {
+            "nl.nldesignsystem.css.property": {
+              "syntax": "<color>",
+              "inherits": true
+            },
+            "nl.nldesignsystem.fallback": ["utrecht.alert.border-color"]
+          }
+        },
+        "border-width": {
+          "$extensions": {
+            "nl.nldesignsystem.css.property": {
+              "syntax": "<length>",
+              "inherits": true
+            },
+            "nl.nldesignsystem.fallback": ["utrecht.alert.border-width"]
+          }
+        }
+      },
```

Comment:
Is het echt nodig om de border-width aan te passen voor info?

Reply:
Border-width is al een ondersteunde property in css voor alle varianten. De hele info variant had alleen geen token documentatie.